assets/base.css

```diff
@@ -2303,15 +2303,15 @@ details[open] > .header__icon--menu .icon-hamburger {
 }
 
 .header__submenu.list-menu--disclosure {
-  border-radius: var(--popup-drawer-corner-radius);
-  border-color: rgba(var(--color-foreground), var(--popup-drawer-border-opacity));
+  border-radius: var(--popup-corner-radius);
```

Comment:
Not in scope for this PR, but just wanted to make a note.

We apply essentially the same set of CSS properties in multiple files for all "popups" and "drawers". Is there an opportunity to create (and add to the HTML) a utility class  like `.popup` and `.drawer` on which we apply the CSS? Thoughts on this approach vs. the current? This pattern would apply to other settings as well.

Reply:
Yes absolutely. I'd much prefer to utilize new global classes wherever we can. In a perfect world, global settings would change styles at the highest level with only overrides/exceptions taking place in lower level section styles.